src/app/project-links.ts

```diff
@@ -13,7 +13,7 @@ export default class ProjectLinks {
     public static readonly URL_SETTINGS: string = "v/settings";
     public static readonly URL_ADMIN_DASHBOARD: string = "v/admin-dashboard";
     public static readonly URL_FLOW_DETAILS: string = "flow-details";
```

Comment:
I guess should be `/v/flow-details` too

Reply:
Fixed

Reply:
Come one, then why the hell it is declared there among with other urls, as if it's from the same group...